scripts/get-prs

```diff
@@ -20,13 +36,93 @@ import sys
 
 from github import Github
 
-test_start_re = "[Tt]est with:?"
-repo_ref_re = "([a-zA-Z0-9_-]+/[a-zA-Z0-9_-]+#[0-9]+)"
-pr_re = re.compile(repo_ref_re)
-repo_sep_re = "([, ]+(and)?[ ]*)"
-test_re = re.compile(f"{test_start_re}[ ]*({repo_ref_re}{repo_sep_re}?)+")
+# Start with a few simple regex'es and re-use them to build more complex ones.
+# Using "(?:...)" makes a regex a non-capturing group, which is basically an
+# atomic expression again that operators can be applied to.
+def regex_atom(regex):
+    return f"(?:{regex})"
 
-if __name__ == '__main__':
+whitespace_re = regex_atom("[ \t]")
```

Comment:
I don't think `[\t]` makes sense here (this is PR descriptions, not text files), and I find `[ ]` much easier to read than `{whitespace_re}` which obscures what is going on in the rest of the expressions. I.e. I think we should remove whitespace_re and use `[ ]` directly instead.